.devcontainer/oncreate

```diff
@@ -6,3 +6,6 @@ echo "Running Azure SDK for Rust devcontainer post create script..."
 
 echo "Updating toolchains..."
 rustup show
```

Comment:
This won't work anymore. While you're in there, can you change this to `rustup install`?